### ISSUE TITLE
problem with install layers

### ISSUE THREAD

**Issue by ysolis** [newrelic/newrelic-lambda-cli#172]
I am installing updateing my layers in AWS CodeBuilld

**buildspec.yml**

```

####################################
#
# Require: ubuntu ec2 instance
#
####################################
version: 0.2

env:
  shell: bash

phases:
  install:
    commands:
    - apt update -y
    - apt install nodejs -y
    - npm install -g serverless
    - npm install
    - pip3 install newrelic-lambda-cli
  build:
    commands:
    - sls deploy --stage $stage --region $region -v
  post_build:
    commands:
      - newrelic-lambda layers install --function all --nr-account-id XXXXXXX --nr-api-key NRAK-XXXXXXXXXXXXXXXXX --upgrade --aws-region $region
      
```

this is failing in the CodeBuild process, in the post_build:

```
Traceback (most recent call last):
  File "/root/.pyenv/versions/3.8.8/bin/newrelic-lambda", line 5, in <module>
    from newrelic_lambda_cli.cli import main
  File "/root/.pyenv/versions/3.8.8/lib/python3.8/site-packages/newrelic_lambda_cli/cli/__init__.py", line 5, in <module>
    from newrelic_lambda_cli.cli import functions, integrations, layers, subscriptions
  File "/root/.pyenv/versions/3.8.8/lib/python3.8/site-packages/newrelic_lambda_cli/cli/layers.py", line 8, in <module>
    from newrelic_lambda_cli import layers, permissions
  File "/root/.pyenv/versions/3.8.8/lib/python3.8/site-packages/newrelic_lambda_cli/layers.py", line 5, in <module>
    import enquiries
ModuleNotFoundError: No module named 'enquiries'

```

it's a weird error because was working up to 3 days ago

**Comment by kolanos** [comment]
@ysolis Thanks for reporting. We'll get a fix out ASAP. In the meantime, can you downgrade to the previous CLI version?

**Comment by kolanos** [comment]
@ysolis Ok, this should be fixed in `v0.6.1`. Let us know if you run into any other issues. Thanks again.
